### ISSUE TITLE
Restructure to better explain hosted and packaged applications

### ISSUE THREAD

**Issue by jplyle** [sysapps/runtime#9]
The current specification defines what a 'hosted' application is within the 'packaged applications' section.  It would be better to introduce both in an earlier section, and then have sections for each, specifying in more detail.

I also think this section should come earlier in the document, because many other sections (including the manifest section) change depending on whether it is a packaged or hosted application.

I am happy to help write / review / comment on any changes.

**Comment by marcoscaceres** [comment]
I agree (see my [rewrite of the abstract](https://github.com/sysapps/runtime/pull/3)). 

I've started [early work](http://marcoscaceres.github.com/sysapps/proposals/RunTime-Security/Overview.html) on breaking the two up this already - actually, it's broken into more parts than that. 

How I see it is:
1. manifest:
   - Internationalization model. 
   - processing rules for each manifest member + how to acquire the manifest over HTTP. 
2. hosted Web apps.
   - security model. 
   - Author facing APIs. 
   - Auto discovery.  
3. Packaged apps
   - format (zip) and dealing with all the edge cases. 
   - how to acquire the manifest in a package.
   - security model + app://  

We need to have a serious discussion if we really need 3.

**Comment by jplyle** [comment]
Ok.  Your changes to the abstract are a great improvement.

One of the tasks I had intended to do this week was to suggest some wording improvements to Section 9.4, as well.  Should I wait for your changes?  I'm also, as discussed on the mailing lists, planning to update and improve the "Security Requirements for System Applications" text I wrote in January ( http://sysapps.github.com/sysapps/proposals/SecurityModel/RequirementsForSecurityModel.html ).  I don't know whether this can find a good home somewhere in this repository.

Thanks,

**Comment by marcoscaceres** [comment]
@jplyle no, please don't wait. I'm currently only focusing on processing the manifest and the i18n model, so there is no risk of overlap. However, is there some way we can coordinate this process a bit better? It would be good to know what section each person is working on and for use to be able to communicate in realtime. Are you on IRC? if so, I'm always on #sysapps on the w3c's IRC server.

**Comment by jplyle** [comment]
@marcoscaceres Sure.  I've joined the channel.
